src/toil_scripts/gatk_germline/germline.py

```diff
@@ -1,688 +1,785 @@
+
     # Run subparser
-    parser_run = subparsers.add_parser('run', help='Runs the GATK germline pipeline')
-    group = parser_run.add_mutually_exclusive_group(required=True)
-    parser_run.add_argument('--config', default='gatk_germline.config', type=str,
-                            help='Path to the (filled in) config file, generated with "generate-config".')
-    group.add_argument('--manifest', default='gatk-germline-manifest.tsv', type=str,
-                       help='Path to the (filled in) manifest file, generated with "generate-manifest". '
-                            '\nDefault value: "%(default)s".')
-    group.add_argument('--sample', default=None, nargs='2', type=str,
-                       help='Space delimited sample UUID and BAM file in the format: uuid url')
-    parser_run.add_argument('--output-dir', default=None, help='Full path to directory or filename where '
-                                                               'final results will be output')    
-    parser_run.add_argument('-s', '--suffix', default='.bqsr', help='Additional suffix to add to the names of the output files')
+    parser_run = subparsers.add_parser('run',
+                                       help='Runs the GATK germline pipeline')
+    parser_run.add_argument('--config',
+                            required=True,
+                            type=str,
+                            help='Path to the (filled in) config file, generated with '
+                                 '"generate-config".')
+    parser_run.add_argument('--manifest',
+                            type=str,
+                            help='Path to the (filled in) manifest file, generated with '
+                                 '"generate-manifest". '
+                                 '\nDefault value: "%(default)s".')
+    parser_run.add_argument('--sample',
+                            default=None,
+                            nargs=2,
+                            type=str,
+                            help='Space delimited sample UUID and BAM file in the format: uuid url')
+    parser_run.add_argument('--output-dir',
+                            default=None,
+                            help='Path/URL to output directory')
+    parser_run.add_argument('-s', '--suffix',
+                            default=None,
+                            help='Additional suffix to add to the names of the output files')
+    parser_run.add_argument('--preprocess-only',
+                            action='store_true',
+                            help='Only runs preprocessing steps')
     Job.Runner.addToilOptions(parser_run)
     options = parser.parse_args()
 
     cwd = os.getcwd()
-    if options.command == 'generate-config' or options.command == 'generate':
-        generate_file(os.path.join(cwd, 'gatk-preprocessing.config'), generate_config)
+    if options.command == 'generate-inputs' or options.command == 'generate':
```

Comment:
Why change the convention from every other pipeline to `generate-inputs` over `generate-config` ? Your function is even still called `generate_config()`